crates/sui-framework/sources/governance/sui_system.move

```diff
@@ -578,7 +578,7 @@ module sui::sui_system {
     /// 1. Add storage charge to the storage fund.
     /// 2. Burn the storage rebates from the storage fund. These are already refunded to transaction sender's
     ///    gas coins.
-    /// 3. Distribute computation charge to validator stake and delegation stake.
+    /// 3. Distribute computation charge to validator stake and stake stake.
```

Comment:
stake stake 🤣 but let's land the PR first and fix this later

Reply:
Changed this at the same time as I was reverting the `safe_tests` file.